### DISCUSSION TITLE
Playlist queue error (?)

### DISCUSSION THREAD

**Discussion by VenansiusShn** [eritislami/evobot#840] [Q&A] [answered]
I am currently experiencing a little problem with my evobot. I like to use playlists for music. But everytime i put playlist on the queue, it only fits 10 songs (out of 74 songs) from my playlist. Any idea/advice how to solve it? please? Thank you

**Comment by eritislami** [comment, marked as answer]
https://github.com/eritislami/evobot/blob/42c2352de12ec2ce6080d93a1197cadb1a72de5d/config.json.example#L5

Did you check the config key?

**Reply by VenansiusShn** [reply]
Oh God, thank you very much, im sorry for my stupid question ._.
